reference/matrix/batch_csr_kernels.cpp

```diff
@@ -9,26 +9,23 @@
 #include <ginkgo/core/base/batch_multi_vector.hpp>
 #include <ginkgo/core/matrix/batch_csr.hpp>
 
+
+#define GKO_DEVICE_NAMESPACE reference
```

Comment:
The OMP files don't have this. Is it set by CMake for OMP? If so, maybe also set it via CMake here.

Reply:
These are defined in `common/unified/base/kernel_launch.hpp` for the rest of the executors, but not defined for reference.

Reply:
They should be removed there, since they (should) already get set via CMake for other files, e.g. cuda_hip unified files.